### ISSUE TITLE
Wrong about test

### ISSUE THREAD

**Issue by ChaosNN** [davidtvs/PyTorch-ENet#33]
The model I train and you provide both have this problem.
>>>> Avg. loss: 0.0000 | Mean IoU: nan
unlabeled: nan
road: nan
sidewalk: nan
building: nan
wall: nan
fence: nan
pole: nan
traffic_light: nan
traffic_sign: nan
vegetation: nan
terrain: nan
sky: nan
person: nan
rider: nan
car: nan
truck: nan
bus: nan
train: nan
motorcycle: nan
bicycle: nan

**Comment by davidtvs** [comment]
if you ran cityscapes in test mode that's expected since the test set of cityscapes is private; therefore, there are no images

**Comment by ZeroCool11000** [comment]
So how to test on another test set after training ?

**Comment by davidtvs** [comment]
You can test on CamVid, but for other datasets you have to do your own dataloader and the test script (maybe reusing the [test function](https://github.com/davidtvs/PyTorch-ENet/blob/master/main.py#L217) and/or [test class](https://github.com/davidtvs/PyTorch-ENet/blob/master/test.py)).

**Comment by ZeroCool11000** [comment]
Ok thank you, kind regards.
